README.md

```diff
@@ -20,38 +30,31 @@ Then:
 - Create your test safe
```

Comment:
```suggestion
- Create your [test safe](https://rinkeby.gnosis-safe.io) 
```

Reply:
yeah seems good, i'll commit both of these suggestions.